cmake/Buildgym-ignition.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2021 iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Diego Ferigo <diego.ferigo@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(iDynTree QUIET)
+
+ycm_ep_helper(gym-ignition TYPE GIT
+              STYLE GITHUB
+              REPOSITORY robotology/gym-ignition.git
+              TAG master
+              COMPONENT core
+              FOLDER src
+              CMAKE_ARGS -DSCENARIO_USE_IGNITION:BOOL=${ROBOTOLOGY_USES_IGNITION}
+                         -DSCENARIO_ENABLE_BINDINGS:BOOL=${ROBOTOLOGY_USES_PYTHON}
+                         -DBINDINGS_INSTALL_PREFIX:PATH=${ROBOTOLOGY_SUPERBUILD_PYTHON_INSTALL_DIR}
+              DEPENDS iDynTree ign-gazebo5)
```

Comment:
@traversaro is the `ign-gazeboX` dependency OK? (There's a typo and this should be 4 for the current `master`, but I'm not sure that adding it is necessary)

Reply:
I typically set `DEPENDS`  only to the value of other projects that are handled by the superbuild. I am not really sure how it works for projects not handled by the superbuild, so if it works I would just leave iDynTree there.

Reply:
Remove in https://github.com/robotology/robotology-superbuild/compare/6a3e8b6b3ff38756781c575274145dc19f5aff8b..b76fbfe3a2dc0529aa5d7e9dc32dcced200f03fa